File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// WaterQualityArchive SDK exists test

require_once __DIR__ . '/../waterqualityarchive_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = WaterQualityArchiveSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: lua/test/exists_test.lua
-- WaterQualityArchive SDK exists test

local sdk = require("water-quality-archive_sdk")

describe("WaterQualityArchiveSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
